compute/apiv1/smoke_test.go

```diff
@@ -396,3 +452,125 @@ func TestPaginationMapResponseMaxRes(t *testing.T) {
 		t.Error("Couldn't find the accelerator in the response")
 	}
 }
+
+func TestTypeInt64(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping smoke test in short mode")
+	}
+	ctx := context.Background()
+	c, err := NewImagesRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	space := uid.NewSpace("gogapic", nil)
+	name := space.New()
+
+	codes := []int64{
+		5543610867827062957,
+	}
+	req := &computepb.InsertImageRequest{
+		Project: projectId,
+		ImageResource: &computepb.Image{
+			Name:         &name,
+			LicenseCodes: codes,
+			SourceImage:  proto.String("projects/debian-cloud/global/images/debian-10-buster-v20210721"),
+		},
+	}
+	globalClient, err := NewGlobalOperationsRESTClient(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	insert, err := c.Insert(ctx, req)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = globalClient.Wait(ctx,
+		&computepb.WaitGlobalOperationRequest{
+			Project:   projectId,
+			Operation: insert.Proto().GetName(),
+		})
+	if err != nil {
+		t.Error(err)
+	}
+
+	defer func(c *ImagesClient, ctx context.Context, req *computepb.DeleteImageRequest) {
```

Comment:
Are the `func` params necessary? They should be captured in the closure if they are defined in the lines preceding this `defer`

Reply:
done! didn't expect that params would be captured.